modules/special-rules/nav.adoc

```diff
@@ -18,43 +18,60 @@
   *** xref:vortex-missile.adoc[]
 
  ** xref:special-abilities.adoc[]
-  *** xref:anti-personnel-ability.adoc[]
-  *** xref:artillery.adoc[]
+
+ ** Special abilities: unit design modifiers
+  *** xref:armour-1.adoc[]
   *** xref:assault.adoc[]
-  *** xref:bridge.adoc[]
-  *** xref:brood-telepathy.adoc[]
-  *** xref:cavalry.adoc[]
-  *** xref:close-support.adoc[]
-  *** xref:deathwing.adoc[]
-  *** xref:engineers.adoc[]
+  *** xref:assault-1.adoc[]
   *** xref:fast.adoc[]
-  *** xref:flak.adoc[]
+  *** xref:firepower-1.adoc[]
   *** xref:heavy-weapons.adoc[]
   *** xref:hero.adoc[]
-  *** xref:hit-and-run.adoc[]
+  *** xref:long-range.adoc[]
+  *** xref:self-propelled-artillery.adoc[]
+  *** xref:shootier.adoc[]
+  *** xref:siege.adoc[]
+
+ ** Special abilities: unit abilities
+  *** xref:artillery.adoc[]
+  *** xref:cavalry.adoc[]
+  *** xref:close-support.adoc[]
+  *** xref:flak.adoc[]
+  *** xref:hitch-x.adoc[] *NEW* equivalent of Transport (X)
+  *** xref:holo-fields.adoc[]
+  *** xref:ignore-terrain.adoc[]
   *** xref:infiltrators.adoc[]
   *** xref:jump-packs.adoc[]
   *** xref:kroot.adoc[]
-  *** xref:long-range.adoc[]
-  *** xref:medic.adoc[]
-  *** xref:open-top.adoc[]
-  *** xref:ordo-malleus.adoc[]
   *** xref:psyker.adoc[]
   *** xref:rampage.adoc[]
   *** xref:rapid-fire.adoc[]
-  *** xref:ravenwing.adoc[]
-  *** xref:recovery.adoc[]
   *** xref:save.adoc[]
-  *** xref:self-propelled-artillery.adoc[]
-  *** xref:shootier.adoc[]
-  *** xref:siege.adoc[]
   *** xref:skimmer.adoc[]
-  *** xref:space-wolves.adoc[]
-  *** xref:stubborn.adoc[]
   *** xref:supreme-commander.adoc[]
-  *** xref:tank-destroyer.adoc[]
+  *** xref:synapse-creature.adoc[]
   *** xref:transport-x.adoc[]
   *** xref:tunneller.adoc[]
+  *** xref:war-engine-supreme-commander.adoc[] *NEW* this is just the ability to re-roll when supporting, for Dominatrixes and maybe add to Gargant Warlords
```

Comment:
This is a bit of a mouthful. I'm not intimate with the drive behind it, but rather than add a new ability could we just clarify the Supreme Commander rule for the war engine case instead? Or is that problematic for some reason?

Reply:
As per the note, it **doesn't** currently apply to Ork Supreme Commanders in Gargants.

We could add it to the Supreme Commander rule, rather like I added a clause for war engines to the Save page, but it would definitely need to be discussed with players first, and even it's accepted we'd need another 'this is not how the rule originally worked, discuss amongst yourselves' edition note.